src/color/readableColor.js

```diff
@@ -1,8 +1,15 @@
 // @flow
 import getLuminance from './getLuminance'
+import meetsContrastGuidelines from './meetsContrastGuidelines'
 
 /**
- * Returns black or white (or optional light and dark return colors) for best contrast depending on the luminosity of the given color.
+ * Returns black or white (or optional light and dark return colors) for best
+ * contrast depending on the luminosity of the given color.
+ *
+ * Set `strict` to `true` to ensure that the return color always meets or
+ * exceeds WCAG level AA or greater. In this case, black or white are used as
+ * fallback colors.
```

Comment:
Let's update this to make it clear that this only applies when passing custom colors.